README.md

```diff
@@ -34,3 +34,48 @@ system:
   forwardingExtensionName: vRouter forwarding extension
 ...
 ```
+## Clear-ComputeNode.ps1
+
+This script:
+
+1. Uninstalls all components of Tungsten Fabric.
+2. Removes all services created during installation.
+3. Uninstalls all docker images, removes all containers and Docker networks.
+4. Removes all HNS networks.
+5. Removes installation directory and directories with logs and configuration files.
+
+All operations are executed on a local machine.
+
+Example invocation:
+```
+.\Clear-ComputeNode.ps1 `
+    -AdapterName "Ethernet1" `
+    -ForwardingExtensionName "vRouter forwarding extension" `
+    -VMSwitchName "Layered Ethernet1" `
+    -ConfigAndLogDir "C:\ProgramData\Contrail" `
+    -InstallationDir "C:\Program Files\Juniper Networks"
+```
+
+All arguments are optional. Default values are demonstrated in the example above.
+
+Arguments `AdapterName`, `ForwardingExtensionName` and `VMSwitchName` refer to: name of physical interface controller by vRouter, name of forwarding extension and name of virtual switch created by vRouter.
```

Comment:
`s/controller/controlled`

> virtual switch created by vRouter

`used by vRouter` or `where vRouter is installed`